aircon/src/main/java/com/ironsource/aura/aircon/utils/AirConUtils.java

```diff
@@ -41,7 +41,14 @@ public static ColorInt hexToColorInt(String colorInHex) {
             }
         }
 
-        return null;
+	    try {
+			log().e("Parsing default color hex: " + defaultValue);
+		    return new ColorInt(Color.parseColor(defaultValue));
+	    } catch (Exception e) {
+		    log().e("Failed to parse default color hex: " + defaultValue);
+		    log().logException(e);
+	    }
```

Comment:
This duplication can be easily resolved. Please move this code to a separate method.

Reply:
right, fixed